lib/winrm-fs/core/file_transporter.rb

```diff
@@ -67,17 +75,16 @@ def initialize(shell, opts = {})
         # * progress yields block like net-scp progress
         # * final API: def upload(locals, remote, _options = {}, &_progress)
         #
-        # @param locals [Array<String>,String] one or more local file or
-        #   directory paths
+        # @param locals [Array<String>,String,StringIO] one or more
+        #   local file or directory paths, StringIO objects also accepted
```

Comment:
Probably should clarify only a single StringIO object is valid.

Reply:
I think we are good here since it explicitly states `StringIO` and not `Array<StringIO>`